SynthDet/Assets/Scripts/Randomizers/LightingInfoMetricReporter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using SynthDet.RandomizerTags;
+using UnityEngine;
+using UnityEngine.Perception.GroundTruth;
+using UnityEngine.Perception.Randomization.Randomizers;
+
+namespace SynthDet.Randomizers
+{
+    /// <summary>
+    /// Reports metrics for light intensity, light colour, and rotation for objects that contain a light component and are tagged with <see cref="LightingInfoMetricReporterTag"/>.
+    /// </summary>
+    [Serializable]
+    [AddRandomizerMenu("Perception/My Lighting Info Metric Reporter")]
+    public class LightingInfoMetricReporter : Randomizer
```

Comment:
Do we really want to suggest people use tags for this? For lighting, this could easily be done in a MonoBehaviour

Reply:
<resolved offline>
Summary: The Randomizer workflow provides a powerful way of coordinating timing of code execution within the Scenario, and thus makes it easy to make sure the reporting is done after all other Randomizers are done. Thus, for the time being, we will keep this pattern for metric reporters. This may be iterated on in future versions.